javascript/grammar.json

```
@@ -232,6 +232,61 @@
           }
         }
       },
+      "hashbang_comments": {
+        "__compat": {
+          "description": "Hashbang (<code>#!</code>) comment syntax",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Lexical_grammar#Hashbang_comments",
+          "support": {
+            "chrome": {
+              "version_added": "74"
+            },
+            "chrome_android": {
+              "version_added": "74"
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "67"
+            },
+            "firefox_android": {
+              "version_added": "67"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "nodejs": {
+              "version_added": true
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "samsunginternet_android": {
+              "version_added": null
```

Looks good to me!  Just got one little update to make:

```suggestion
              "version_added": true
```